cmd/cmd_suite_test.go

```diff
@@ -71,19 +71,25 @@ func (f tmpFile) write(fileContent string) error {
 	return err
 }
 
-func openTmpFile(path string) (tmpFile, error) {
+func openTmpDir(prefix string) (string, error) {
```

Comment:
I don't think `prefix` is ever used here - copy + paste bug?

Reply:
It's used here:
https://github.com/CircleCI-Public/circleci-cli/pull/23/files/1c600eaec59f3c63688668b38f7e6753e88de3bb#diff-bee1de2bffa142cede85d49cea1a90f7R76

So you can create a temp directory without specifying the prefix, just do `openTmpDir("")`.